src/github/handlers/push-event.ts

```diff
@@ -11,14 +11,15 @@ function constructErrorBody(
   errors: Iterable<ValueError> | (YAML.YAMLError | ValueError)[],
   rawData: string | null,
   repository: GitHubContext<"push">["payload"]["repository"],
-  after: string
+  after: string,
+  configPath: string
```

Comment:
This seems like the most key change here, but I don't understand what his value is supposed to be compared to what it used to be